streaming/src/test/scala/org/apache/spark/streaming/CheckpointSuite.scala

```diff
@@ -56,7 +172,7 @@ class CheckpointSuite extends TestSuiteBase {
 
   override def afterFunction() {
     super.afterFunction()
-    if (ssc != null) ssc.stop()
+    StreamingContext.getActive().foreach { _.stop() }
```

Comment:
If SparkContext is created in the StreamingContext's constructor, but `StreamingContext's.stop` is not called, the line cannot stop the `SparkContext`.